README.md

```diff
@@ -902,9 +915,11 @@ Table of Contents
 
 ## Issue Tracking and Project Management
 
+   * [acunote.com](https://www.acunote.com/) — Free project management and SCRUM software for up to 5 team members
    * [acunote.com](https://www.acunote.com/) — Free project management and SCRUM software for up to 5 team members
```

Comment:
This line is duplicated

Reply:
Yup I noticed. I think it's my editor's mistake.